.github/workflows/smoke-tests.yml

```diff
@@ -0,0 +1,73 @@
+name: smoke-tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      smoketest_versions:
+        description: Set SMOKETEST_VERSIONS environment variable
+        default: latest
+  push:
+    branches:
+      - main
+
+concurrency:
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
```

Comment:
I'm not sure if we want that behaviour, I'd say for PRs might make sense, but for merge commits, I'd prefer to keep them running, so every commit will have a smoke test ideally.

Maybe a timeout could be a good one to be added instead, what do you think?

Reply:
I added this because the smoke tests cannot run in parallel and will fail with terraform resource conflicts.

Reply:
Gotcha, what about removing the `cancel-in-progress`?

Reply:
Ah, so the latter job will wait in queue until the first one is finished. is this correct?

Reply:
https://github.com/elastic/apm-aws-lambda/pull/370/commits/ba2035694af283162e34f70d9b05ee4a6525daba

Reply:
That's my understanding as per [concurrency](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#concurrency)

Reply:
> so every commit will have a smoke test ideally.

Also in Jenkins, this was running only when triggered manually.